src/plugins/intel_gna/src/backend/gna_limitations.cpp

```diff
@@ -695,18 +691,27 @@ size_t Limitations::get_min_batch_to_fit_in_buffer(InferenceEngine::DataPtr inpu
     auto total_size = InferenceEngine::details::product(std::begin(input->getDims()), std::end(input->getDims()));
     return total_size / kBufferMaxSize + 1;
 }
+template <typename T, typename U>
+U GetValueForKey(const T& key, const std::unordered_map<T, U>& mapping) {
```

Comment:
there is already method in commons
https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_gna/src/common/misc_utils.hpp#L11

Reply:
good point